docs/source/models/causes/maternal_disorders/gbd_2021_mncnh/residual_maternal_disorders.rst

```diff
@@ -0,0 +1,324 @@
+be covered as part of the residual maternal disorders cause modeling strategy in this
+document is included in the table below.
+
+.. list-table::
+  :header-rows: 1
+
+  * - Subcause
+    - Modeled?
+    - Documentation link
+  * - Maternal hemorrhage (c_367)
+    - Modeled
+    - :ref:`Maternal hemorrhage document <2021_cause_maternal_hemorrhage_mncnh>`
+  * - Maternal sepsis and other maternal infections (c_368)
+    - Modeled
+    - :ref:`Maternal sepsis document <2021_cause_maternal_sepsis_mncnh>`
+  * - Maternal hypertensive disorders (c_369)
+    - **Unmodeled** (will eventually be modeled in wave III)
+    - Covered in this document (:ref:`Future documentation page for hypertensive disorders <2021_cause_maternal_hypertension_mncnh>`)
+  * - Maternal obstructed labor and uterine rupture (c_370)
+    - Modeled
+    - :ref:`Maternal obstructed labor and uterine rupture docs <2021_cause_obstructed_labor_mncnh>`
+  * - Ectopic pregnancy (c_374)
+    - Modeled
+    - Included in the :ref:`partial term pregnancy cause docs <2021_cause_partial_term_pregnancy_causes_mncnh>`
+  * - Maternal abortion and miscarriage (c_995)
+    - Modeled
+    - Included in the :ref:`partial term pregnancy cause docs <2021_cause_partial_term_pregnancy_causes_mncnh>`
+  * - Other direct maternal disorders (c_379)
+    - **Unmodeled**
+    - Covered in this document
+  * - Indirect maternal deaths
+    - **Unmodeled**
+    - Covered in this document
+  * - Late maternal deaths (c_376)
+    - **Unmodeled**
+    - Covered in this document
+  * - Maternal deaths aggravated by HIV/AIDS (c_741)
+    - **Unmodeled**
+    - Covered in this document
+
+Therefore, the residual maternal disorders cause modeling strategy will include
+outcomes due to the causes included in the following table (along with a summary of the estimates
+available for each cause).
+
+.. list-table:: Included causes
+  :header-rows: 1
+
+  * - Cause
+    - Cause ID
+    - YLLs
+    - YLDs
+    - Incidence
+    - Note
+  * - Indirect maternal deaths
+    - 375
+    - True
+    - False
+    - False
+    - 
+  * - Other direct maternal disorders 
+    - 379
+    - True
+    - True
+    - False
+    - 
+  * - Late maternal deaths 
+    - 376
+    - True
+    - False
+    - False
+    - 
+  * - Maternal deaths aggravated by HIV/AIDs
+    - 741
+    - True
+    - False
+    - False
+    - 
+  * - Maternal hypertensive disorders
+    - 369
+    - True
+    - True
+    - True
+    - Note this cause will eventually be excluded from the residual maternal disorders cause model
+
+Summary of modeling strategy
+++++++++++++++++++++++++++++
+
+We will assume a residual maternal disorders incidence ratio equal to 1 for all "full term" pregnancies that 
+result in live and still births in our simulation. This will easily allow us to apply both mortality and 
+morbidity due to residual maternal disorders to this population without having to directly model residual
+maternal disorders incident cases.
```

Comment:
I thought I would see why we did this when I got to this cause model, but it's still unclear to me: what is the advantage of representing this in our cause model? Why not just exclude the `1 - ir ` arrow completely? We could change "affected with residual maternal disorders" to be "apply YLDs due to residual maternal disorders" which I think would be conceptually clearer.

Reply:
Thanks for all your feedback! I agree that "apply YLDs due to residual maternal disorders" is a better title, I will update that. And I basically made the decision to structure the cause model this way so that it would have the same underlying structure as the other maternal disorders causes which I thought may be helpful in implementation by being able to reuse the same custom component. I'll plan to bring this up in the RT/SE check-in today to see if that is actually useful and if not then I will update to remove the `1 - ir` arrow because I agree that is better otherwise!